src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -108,6 +108,31 @@ public static void Add_CookieCollection_Invalid()
             Assert.Throws<ArgumentNullException>(() => cc.Add((CookieCollection)null));
         }
 
+        [Fact]
+        public static void Clear_Success()
+        {
+            ICollection<Cookie> cc = CreateCookieCollection1();
+            cc.Clear();
+            Assert.Equal(0, cc.Count);
+        }
+
+        [Fact]
+        public static void Contains_Success()
+        {
+            ICollection<Cookie> cc = new CookieCollection();
+            cc.Add(c1);
+            Assert.True(cc.Contains(c1));
+            Assert.False(cc.Contains(c2));
+        }
+
+        [Fact]
+        public static void Remove_Success()
+        {
+            ICollection<Cookie> cc = CreateCookieCollection1();
+            Assert.True(cc.Remove(c1));
+            Assert.False(cc.Contains(c1));
```

Comment:
Nit: it'd be good to validate other impacted members as well, e.g. that the count has appropriately decreased by 1.